### PR TITLE
New rigellian plague

### PR DESCRIPTION
**Pull request by Zitchas**
**Content (Missions / Jobs)**

## Summary
Adds a small flavour mission where the player gets to help stop a plague (or die, depending on choices). I'll be adding one additional intermediary mission so that it isn't quite as much a "refuse once and you are dead" thing. And possibly reworking the choices a bit to give the player the chance to go back. 

## PR Checklist
 - [X] ~I updated the copyright attributions, or decline to claim copyright of any assets produced or modified~
 - [x] ~I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}~
 - [x] ~I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~
  
  ## PR Intention:
I'm intending on building this up to the point where it has three missions and a repeatable job. Depending on player choices, I'm envisioning three potential paths:
1) mission(accept), mission(accept), complete 4x jobs, mission (current outbreak contained)
2) mission(refuse), mission(accept), complete 4x jobs, (current outbreak contained but wider effect)
3) mission(refuse), mission(refuse), player killed.

Basically ensuring that the player gets at least two chances to prevent dying of the plague. 

There's also the potential of adding a fourth mission to the refuse/refuse chain just to prolongue the agony over a couple missions. Also potentially looking for a "greyer" 4th option. Probably something to the effect of if the player goes the refuse/refuse route, they can decide to skip town and stay out of human space for a  year or so until the plague timer runs out, in which case a bunch of planets suffer but the player gets to avoid dying without getting vaccinated.

### CODE REVIEW

**Comment by Amazinite** [comment]
Been wondering when you'd PR this one. :)

File: data/transport missions.txt [hunk 1]
```diff
@@ -3469,3 +3469,87 @@ mission "Inheritance Redirection"
...
+mission "The New Rigellian Plague Spreading"
+	landing
+	invisible
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	to offer
+		has "event: plague spreading"
+	on enter
+		conversation
+			`... After a series of hacking coughs, you finally collapse. Without enough air in you, your limbs begin to numb. In, out, in, out, but it is of no use. You try and drag yourself to your ships panel to send a distress signal but your arms gave out beforehand, leaving you wanting upon the floor. You make a scream, scared and wanting someone, anyone to help you. It is of no use, you could barely get the noise out of you before a wheeze cut it short. Out of oxygen, your vision leaves you. All that remains to you is the hum of the ship and the feel of yourself upon the cold floor. Eventually, the feel of cold goes away. It is neither cold nor warm. As you begin to drift out of consciousness, you think of your life up until now; the journey you've taken. You think of your family back on New Boston waiting for you to make a return. All is quiet.`
```
Comment (petervdmeer) [inline data/transport missions.txt]: The player will never see the mission title, only the dialog. Maybe add a reference to the Rigellian Plague to the beginning of this text? (To differentiate this conversation from another missions where the player was just poisoned on the planet he/she left.)

**Comment by Arrow2thekn33** [comment]
I'm not entirely sure how the code works or I'd fix it myself, but in the first mission, after accepting the vaccine and agreeing to transport them, this popped up.
rigellianplagueerror

File: data/human/transport missions.txt [hunk 2]
```diff
@@ -3570,3 +3570,87 @@ mission "Hauler VI cargo prototype 3"
...
+			`	As the line moves forward, you can see a banner stating "Red Cross Vaccination Clinic - Free Rigellian Plague Vaccinations." The nurses are surprisingly fast, vaccinating people and giving them a card with information at a fast clip. Before you can even make a decision, you are at the front of the line and a nurse is standing in front of you scanning your ID.`
+			`	"Good morning Captain! I see your medical records indicate you haven't received the vaccine yet," she pauses, skimming a few lines on her scanner. "Ah, good. You aren't allergic to anything in this vaccine, and haven't had any reactions to medicines in the past. This vaccine is designed to boost your immune system against three variants of the Rigellian Plague. It won't guarantee your immunity, but it will significantly improve your resistance to it. In which arm would you prefer to receive the vaccine?"`
```
Comment (Arrow2thekn33) [inline data/human/transport missions.txt]:
```suggestion
			`	"Good morning Captain! I see your medical records indicate you haven't received the vaccine yet," she pauses, skimming a few lines on her scanner. "Ah, good. You aren't allergic to anything in this vaccine, and haven't had any reactions to medications in the past. This vaccine is designed to boost your immune system against three variants of the Rigellian Plague. It won't guarantee your immunity, but it will significantly improve your resistance to it. In which arm would you prefer to receive the vaccine?"`
```

File: data/human/transport missions.txt [hunk 3]
```diff
@@ -3570,3 +3570,87 @@ mission "Hauler VI cargo prototype 3"
...
+
+mission "The New Rigellian Plague Discovered"
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	to offer
+		has "event: plague discovered"
+	on offer
+		conversation
+			`The crowd in the spaceport cantina are worriedly watching a news broadcast where the announcer has just finished listing off a number of planets. "Health authorities have stated that the current symptoms point to the first case of the Rigellian plague in over a century. Officials are currently crediting a recent vaccination push by independent humanitarian groups for preventing this from becoming a full fledged pandemic. We will have more updates after this message from our sponsor."`
```
Comment (Arrow2thekn33) [inline data/human/transport missions.txt]:
```suggestion
			`The crowd in the spaceport cantina are worriedly watching a news broadcast where the announcer has just finished listing off a number of planets. "Health authorities have stated that the current symptoms point to the first case of the Rigellian plague in over a century. Officials are currently crediting a recent vaccination push by independent humanitarian groups for preventing this from becoming a full-fledged pandemic. We will have more updates after this message from our sponsor."`
```

File: data/human/transport missions.txt [hunk 4]
```diff
@@ -3570,3 +3570,87 @@ mission "Hauler VI cargo prototype 3"
...
+			`	The crowd disperses back to their tables and you can hear a variety of murmurs of people talking about the news and wondering where it might show up next.`
```
Comment (Arrow2thekn33) [inline data/human/transport missions.txt]:
```suggestion
			`	The crowd disperses back to their tables and you can hear a variety of murmurs of people talking about the news and wondering where the plague might show up next.`
```

File: data/human/transport missions.txt [hunk 5]
```diff
@@ -3570,3 +3570,87 @@ mission "Hauler VI cargo prototype 3"
...
+mission "The New Rigellian Plague Spreading"
+	landing
+	invisible
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	to offer
+		has "event: plague spreading"
+	on enter
+		conversation
+			`... After a series of hacking coughs, you finally collapse. Without enough air in you, your limbs begin to numb. In, out, in, out, but it is of no use. You try and drag yourself to your ships panel to send a distress signal but your arms gave out beforehand, leaving you wanting upon the floor. You make a scream, scared and wanting someone, anyone to help you. It is of no use, you could barely get the noise out of you before a wheeze cut it short. Out of oxygen, your vision leaves you. All that remains to you is the hum of the ship and the feel of yourself upon the cold floor. Eventually, the feel of cold goes away. It is neither cold nor warm. As you begin to drift out of consciousness, you think of your life up until now; the journey you've taken. You think of your family back on New Boston waiting for you to make a return. All is quiet.`
```
Comment (Arrow2thekn33) [inline data/human/transport missions.txt]:
```suggestion
			`... After a series of hacking coughs, you finally collapse. Without enough air in you, your limbs begin to numb. In, out, in, out, but it is of no use. You try and drag yourself to your ship's panel to send a distress signal but your arms gave out beforehand, leaving you wanting upon the floor. You make a scream, scared and wanting someone, anyone to help you. It is of no use, you could barely get the noise out of you before a wheeze cut it short. Out of oxygen, your vision leaves you. All that remains to you is the hum of the ship and the feel of yourself upon the cold floor. Eventually, the feeling of cold goes away. It is neither cold nor warm. As you begin to drift out of consciousness, you think of your life up until now; the journey you've taken. You think of your family back on New Boston waiting for you to make a return. All is quiet.`
```

**Comment by Zitchas** [comment]
This is being closed for now. I'd like to re-open it and get it up to merge-quality in the future, but this isn't a priority right now as I focus on Remnant content.
